test/replication-luatest/quorum_master_test.lua

```diff
@@ -1,6 +1,6 @@
 local t = require('luatest')
 local log = require('log')
-local Cluster =  require('test.luatest_helpers.cluster')
+local Cluster =  require('luatest.replica_set')
```

Comment:
why not `cluster` (all lowercase) like in other tests?

Reply:
Do you really want me to fix this? It is not related to this patch. Sometimes people capitalize it to show that it is a 'class'.